api_tests/base/test_filters.py

```diff
@@ -378,13 +378,72 @@ def test_parse_query_params_uses_field_source_attribute(self):
         query_params = {
             'filter[bool_field]': 'false',
         }
-
         fields = self.view.parse_query_params(query_params)
         parsed_field = fields['filter[bool_field]']['bool_field']
         assert parsed_field['source_field_name'] == 'foobar'
         assert parsed_field['value'] is False
         assert parsed_field['op'] == 'eq'
 
+    def test_multiple_values_filter(self):
```

Comment:
Probably you won't need these specific tests with the change I requested, but you might add one for the notification subscriptions.